src/ORM/PaginatedList.php

```diff
@@ -416,6 +416,10 @@ public function NotFirstPage()
      */
     public function LastPage()
     {
+        if ($this->TotalPages() == 0) {
+            return true;
+        }
```

Comment:
This addresses the specific issue description - but doesn't address the actual acceptance criteria.
Consider for example when total pages is say 5, but someone sets their current page (via get vars) to 10. In that case, this will still return false, which (I assume, I haven't tested this) will "[lead] into returning the NextLink when it shouldn't." which is really the main problem we're trying to solve here.

Instead, we should probably adjust the comparison, per the below suggestion.